joomla/joomla-cms
# Fix table breakpoint in Maintenance: Database view

### Summary of Changes
The table with the database info in the Maintenance: Database view is to wide on some breakpoints.

### Testing Instructions
Go to System -> Maintenance - Database
Set screen size to 800px
Refresh the page an see ie table fits

### Actual result BEFORE applying this Pull Request
![Image 2](https://user-images.githubusercontent.com/5610413/137901653-33d011ec-c5e0-4630-9c6f-b2092dd634ee.jpg)

### Expected result AFTER applying this Pull Request
![Image 4](https://user-images.githubusercontent.com/5610413/137904174-00dbeda1-4968-40ec-a868-fa953d033944.jpg)

### Documentation Changes Required
No
